### PR TITLE
brcm: Add new BT & Wifi firmware required by OrangePI3

### PR DESCRIPTION
**Pull request by gatuno**
Hi

I'm currently working in trying to make LibreElec port to Orange PI 3. Is a work in progress try-and-error. Please add this firmware that is needed for bluetooth and wifi Orange PI 3 drivers.

Firmware was obtained from here:
https://github.com/orangepi-xunlong/OrangePiH6_external/tree/master/ap6256

### CODE REVIEW

File: brcmfmac43456-sdio.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+brcmfmac43456-sdio.xunlong,orangepi-3.txt
```
Comment (jernejsk) [inline brcmfmac43456-sdio.txt]: drop this file

**Comment by jernejsk** [comment]
Does bluetooth really work? Last time I checked mainline kernel didn't support H6 RTC, which is needed for 32kHz clock required by WiFi/BT module.

Anyway, I'll get OPi 3 soon too, so we can work on this together.

**Comment by gatuno** [comment]
I will test bluetooth today. I'm not using mainline kernel, I'm using the kernel that Armbian is using.
I think that the "symbolic file" is also needed (not sure, I just put the 4 files in there and wifi started working)

**Comment by jernejsk** [comment]
You know that no external kernels will be accepted? LE uses vanilla kernel (5.1.0 as of now) with additional patches on top. If you port Armbian patches here, then fine. Since OrangePi 3 support is not included in any kernel release, it has to be added in form of patches.

**Comment by gatuno** [comment]
I did not know about the kernel. In first place I want to make a proof of working. I would like to help to port the Armbian patches.

Right now I'm dealing with the famous 'reboot' bug... and the video gpu...

**Comment by gatuno** [comment]
Bluetooth doesn't work, but at least is detected and show as device hci0. The command hciconfig hci0 up doesn't brinup the interface. Wifi at least is working.

**Comment by jernejsk** [comment]
As expected. RTC driver needs to be extended with H6 support. Wifi works good? Can you paste dmesg? If everything seems right, I'll merge BT firmware too, but please remove  brcmfmac43456-sdio.txt because it's not needed.

**Comment by gatuno** [comment]
dmesg output https://gist.github.com/gatuno/cbd48674d60ea1b016260007e955622c

**Comment by jernejsk** [comment]
ok, seems fine. We'll work on H6 RTC driver later.

**Pull request merged**